docs/_renderer.py

```diff
@@ -240,6 +271,10 @@ def read_file(file: str | Path, root_dir: str | Path | None = None) -> FileConte
 
 
 def check_if_missing_expected_example(el, converted):
+    if os.environ.get("SHINY_MODE", "core") == "express":
+        # We aren't done with all of the express examples yet
```

Comment:
Let's make sure to mention to remove this as a part of https://github.com/posit-dev/py-shiny/issues/1080 ? 

```suggestion
        # TODO: remove once we are done porting express examples
```

Reply:
I left a todo task in #1080 to remind us